rules/list-item.js

```diff
@@ -250,7 +250,7 @@ function validateListItemPrefixCasing(prefix, file) {
 	}
 
 	if (!listItemPrefixCaseWhitelist.has(caseOf(firstWord))) {
-		if (!/\d/.test(firstWord) && !/^["'(]/.test(firstWord)) {
+		if (!/\d/.test(firstWord) && !/^["“'(]/.test(firstWord)) {
```

Comment:
Note that the left and right quotes are separate Unicode characters. `“”`

Reply:
Yes, that normal. We should not allow for a description to starts with a closing `”` curved quote. I added a test case for that at: https://github.com/sindresorhus/awesome-lint/pull/127/commits/e318783dc7520afc227816f8f43f67b8d3be7c8f